web/src/components/MoreOptionsDropdown.tsx

```diff
@@ -75,6 +75,16 @@ export function MoreOptionsDropdown({
     };
   }, [reference, shareUrl, summary, share, copyToClipboard, handleTrackShares]);
 
+  const title =
+    id === 'zone'
+      ? t(`more-options-dropdown.zone-title`)
+      : t(`more-options-dropdown.chart-title`);
+
+  const copyLinkText =
+    id === 'zone'
+      ? t(`more-options-dropdown.copy-zone-link`)
+      : t(`more-options-dropdown.copy-chart-link`);
```

Comment:
Just a tiny nitpick, but it would be slightly more efficient to put the conditional inside the t function.